CliFx/CliApplication.cs

```diff
@@ -143,6 +143,50 @@ await applicationSchema
             return 0;
         }
 
+        /// <summary>
+        /// Handle <see cref="CommandException"/>s differently from the rest because we want to
+        /// display it different based on whether we are showing the help text or not.
+        /// </summary>
+        private int HandleCommandException(IReadOnlyList<string> commandLineArguments, CommandException commandException)
```

Comment:
Missing return documentation.

Reply:
That's fine, the return value is obvious and for private methods it's not too important.